src/SOS/Strike/util.cpp

```diff
@@ -4741,6 +4741,7 @@ const char * const DMLFormats[] =
     "<exec cmd=\"!DumpIL /i %s\">%s</exec>",         // DML_IL
     "<exec cmd=\"!DumpRCW -cw /d %s\">%s</exec>",    // DML_ComWrapperRCW
     "<exec cmd=\"!DumpCCW -cw /d %s\">%s</exec>",    // DML_ComWrapperCCW
+    "<exec cmd=\"dps %s L2\">%s</exec>",             // DML_TaggedMemory (hardcoded current size to 2 pointer sizes)
```

Comment:
Is there any way to not hardcode this given we know the size in bytes? The public API has the flexibility of being a Span. Should we maybe not do anything printing DML? The runtime allocates the buffer, but it's a "scratch memory" area, The delegate gets it passed in and they decide how to use it. Also, how is DML used if the feature is for macOS support?

Reply:
> The runtime allocates the buffer, but it's a "scratch memory" area, The delegate gets it passed in and they decide how to use it.

Agreed but that the very scenario that we will likely need to investigate if we get a crash in an Objective-C scenario - get an `object`'s user defined state.

>Also, how is DML used if the feature is for macOS support?

I've no idea. I am relying on SOS's infrastructure here to do the right thing. The current DMLs all use SOS commands in a format I am not sure work on non-WinDBG scenarios - what is the experience there?

Reply:
There's none. If you see a few lines above, the `"<exec cmd=\"~~[%s]s\">%s</exec>", // DML_ThreadID` is a WinDBG specific command of switch to thread by OS ID. I'd just say if this feature is only supported on macOS, this is only going to be valuable if we end up having the macOS crossdac.

Reply:
Currently only macOS has a tagged memory scenario, that is true. However, it is designed and implemented in a way that it could be utilized in any Interop scenario. If, as @mikem8361 mentions, WinDBG is able to read Mach-O DMPs then this will be useful earlier than the interop team using tagged memory on non-macOS platforms.

Reply:
Yeah, macho crossdac is definitely on the radar. We might just need to revisit this DML usage if the interop ever changes, but I am OK leaving this as-is.

Reply:
I can update this to encode the size. I was thinking about it initially but it looked annoying. Taking a look today it actually isn't too bad so I will make it do the right thing now.